uber/model_checks.py

```diff
@@ -25,6 +25,14 @@ def duplicate_admin(account):
                 return 'That attendee already has an admin account'
 
 
+@validation.AdminAccount
+def duplicate_admin(account):
```

Comment:
Since there's already a function called `duplicate_admin`, and I'm pretty sure Python won't let you overload functions, this wouldn't work. You'll want to change your function's name to better reflect what it does, anyway.